.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -1,6 +1,6 @@
 name: Bug Report
 description: File a bug report
-title: ""
```

Comment:
I noticed that too, I'd simply remove it.

Reply:
Let me try that

Reply:
Yep, removing the title worked 🎉 

![image](https://github.com/user-attachments/assets/067ec6ab-967c-453c-a1ad-d4c60bb68773)